Configuration/StandardSequences/python/ValidationCosmics_cff.py

```diff
@@ -10,8 +10,6 @@
 
 from Validation.Configuration.globalValidationCosmics_cff import *
 
-prevalidation = cms.Sequence(globalPrevalidationCosmics)
+prevalidationCosmics = cms.Sequence(globalPrevalidationCosmics * cms.SequencePlaceholder("mix"))
```

Comment:
I think this is because the cosmics validation is not used in reality in any workflow. A similar change was made on everything else that is run by somebody, I just realized it should be changed here as well. I think we should put the cosmics validation in production in Muon-POG relevant workflows (like the superpointing cosmics). It was asked in the past (~1 year ago), I do not know why it has not been done. Anyway this is not relevant for the current PR.